flytekit/models/array_job.py

```diff
@@ -70,13 +70,21 @@ def to_dict(self):
         """
         :rtype: dict[T, Text]
         """
-        return _json_format.MessageToDict(
-            _array_job.ArrayJob(
+        array_job = None
+        if self.min_successes is not None:
+            array_job = _array_job.ArrayJob(
                 parallelism=self.parallelism,
                 size=self.size,
                 min_successes=self.min_successes,
```

Comment:
is it one-of min_successes and min_success_ratio?

Reply:
yes:  https://github.com/flyteorg/flyteidl/blob/091505aae78d935ae55e4d0e46deee5a1e79a22f/protos/flyteidl/plugins/array_job.proto#L20-L29